CartingApp/Bill.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CartingApp
+{
+    public class Bill
+    {
+        const double discountPercentage = 30;
+        public double totalBill;
+        public double totalBillWithDiscount;
+
+        public Bill()
+        {
+            totalBill = 0;
+            totalBillWithDiscount = 0;
+        }
+
+        public void CalculateTotalBill(List<CartItem> cartItemList)
```

Comment:
If you name a class as 'Bill', then you can name it's function as CalculateTotal(), because whenever you'll call this function, calling bill.CalculateTotal() looks better than bill.CalculateTotalBill()